index.html

```diff
@@ -24,20 +24,22 @@
         </nav>
     </header>
 
-    <section class="intro__container">
-        <h1>I'm Mohammed <br /> Glad to see <br />you!</h1>
-        <p>I’m a software developer! I can help you build a product , feature or website Look through some of my
-            work and
-            experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.
-        </p>
-        <a class="lets__connect-btn" href="#">LET’S CONNECT</a>
-        <ul class="social__links-container">
-            <li><a href="twitter.com/"><img src="./images/twitter.svg" alt="Twiiter logo" /></a></li>
-            <li><a href="linkedin.com/"><img src="./images/Linkedin.svg" alt="LinkedIn logo" /></a></li>
-            <li><a href="medium.com/"><img src="./images/medium.svg" alt="Medium logo" /></a></li>
-            <li><a href="githubusercontent.com/"><img src="./images/github.svg" alt="GitHub logo" /></a></li>
-            <li><a href="angellist.com/"><img src="./images/angellist.svg" alt="angellist logo" /></a></li>
-        </ul>
+    <section class="intro">
```

Comment:
Please use min-width media query at any point between `992px` and `1100px`.
for example:  
```
@media screen and (min-width: 1024px) {
/*
.
.
*/
}
```
Your design looks great in width 1440px but there are some issues in the width between `992px` and `1100px`
| Your design | Figma design |
| --- | --- |
| ![screencapture-raw-githack-Mo7amedSMahdi-Personal-Portfolio-desktop-version-index-html-2022-02-17-13_20_33](https://user-images.githubusercontent.com/24703171/154471289-ee7f3fea-0166-40b9-87e5-24c68b69cf90.png) | ![Template 2-Main](https://user-images.githubusercontent.com/24703171/154471489-b7f47d43-868b-4674-b00b-86a426b08c5f.png) |